### PR TITLE
Update Community Cloud docs

### LINKED ISSUE #740: Explain the differences between the email, Google, and GitHub logins for Community Cloud
@nealmcb raised an issue (#726) about the clarity of purpose between the different account types on Streamlit Community Cloud: email, Google, and GitHub.

> You can choose to log in via either a GitHub or Google account when creating a Community Cloud account. (explain why it might matter which one you use).

This has started to be documented with new features but needs to be more comprehensively introduced from the beginning/top of the Community Cloud documentation.

### PR DESCRIPTION
**Pull request by sfc-gh-dmatthews**
[WIP]

## 📚 Context
Since the sunsetting of Enterprise accounts on Streamlit Community Cloud, there remains some language that is out-of-alignment with the current version of Streamlit Community Cloud. This is a general update to bring screenshots and descriptions in line with the current state of affairs.

## 🧠 Description of Changes
* Updated sign-up instructions and better explained the difference between primary and source control identities.
* Removed duplicate images from a prior renaming of public/images/streamlit-cloud to public/images/streamlit-community-cloud
* Image updates to reflect current UI
* Removed workspace analytics
* Rearrangement of navigation
* Revised app dependencies for clarity
* Added Community Cloud quickstart
* Expanded explanation of GitHub authorization both during account creation and managing afterwards
* Revised and updated Share-your-app instructions

<details><summary>Revised (WIP):</summary>

</details>

<details><summary>Previous:</summary>

</details>

## 💥 Impact
- [X] Not small

## 🌐 References
Notion Sprint tasks
* [Clarify account deletion](https://www.notion.so/snowflake-corp/Update-account-deletion-re-organizations-shared-repos-6e2c5bc571864e28b0b26dbd2c85b2ab?pvs=4)
* [Remove workspace analytics](https://www.notion.so/snowflake-corp/Remove-analytics-from-Community-Cloud-docs-8037c6db8cd94cdd8c2194cbeaf2e4f1?pvs=4)
* [Update deployment gifs](https://www.notion.so/snowflake-corp/Update-gifs-of-deployment-on-streamlit-io-a8b746f5b3884ab28bbf445cab3e4929?pvs=4)
* [Document app menu within Community Cloud](https://www.notion.so/snowflake-corp/Docs-for-App-menu-on-Cloud-09a39e61d9694886b36ce95c5201adf8?pvs=4)
* [Update default url](https://www.notion.so/snowflake-corp/Update-Community-Cloud-docs-default-URL-a4ae15f0cec64986ad453ff957dfa33f?pvs=4)
* [Clean up duplicate images](https://www.notion.so/snowflake-corp/Merge-duplicates-in-public-images-for-community-cloud-ac6c1311750d4eb684210a64bdea218d?pvs=4)
* [Remove enterprise language](https://www.notion.so/snowflake-corp/Remove-enterprise-language-from-community-cloud-docs-4c85ee3af9114e279a17829534319993?pvs=4)

closes #740 

**Contribution License Agreement**
By submitting this pull request you agree that all contributions to this project are made under the Apache 2.0 license.

### CODE REVIEW

File: content/streamlit-cloud/get-started/deploy-an-app/index.md [hunk 1]
```diff
@@ -17,20 +17,40 @@ If you want to deploy your app on a different cloud service, check out the [Depl
 
 Streamlit Community Cloud launches apps directly from your GitHub repo, so your app code and dependencies need to be on GitHub before you try to deploy the app. See [App dependencies](/streamlit-community-cloud/get-started/deploy-an-app/app-dependencies) for more information.
```
Comment (jrieke) [inline content/streamlit-cloud/get-started/deploy-an-app/index.md:19]: Maybe add a sentence "Your directory structure should look similar to this:"

File: content/streamlit-cloud/get-started/deploy-an-app/index.md [hunk 2]
```diff
@@ -17,20 +17,40 @@ If you want to deploy your app on a different cloud service, check out the [Depl
...
```
Comment (jrieke) [inline content/streamlit-cloud/get-started/deploy-an-app/index.md:18]: Maybe turn the last sentence into "For more information on how to specify dependencies, see [App dependencies](...)"

File: content/streamlit-cloud/get-started/deploy-an-app/index.md [hunk 3]
```diff
@@ -17,20 +17,40 @@ If you want to deploy your app on a different cloud service, check out the [Depl
...
 
-### Optionally, add a configuration file
+```
+your-repository/
+├── your_app.py
+└── requirements.txt
+```
+
+If you are including any custom [Configuration](/library/advanced-features/configuration) or [Themeing](/library/advanced-features/theming), make sure your config file is saved relative to the root of your repo. Within your repo, your config file should be `.streamlit/config.toml`.
```
Comment (jrieke) [inline content/streamlit-cloud/get-started/deploy-an-app/index.md:26]: Themeing -> Theming

File: content/streamlit-cloud/get-started/deploy-an-app/index.md [hunk 4]
```diff
@@ -17,20 +17,40 @@ If you want to deploy your app on a different cloud service, check out the [Depl
...
+```
+your-repository/
+├── .streamlit/
+│   └── config.toml
+├── your_app.py
+└── requirements.txt
+```
 
-Specifically, you can add a configuration file to the root (top-level) directory of your repo: create a `.streamlit` folder, and then add a `config.toml` file to that folder. E.g., if your app is in a repo called `my-app`, you would add a file called `my-app/.streamlit/config.toml`. Say you want to set the theme of your app to "dark". You would add the following to your `.streamlit/config.toml` file:
+If your main app file is not located at the root of your repository, your folder structure might look like:
```
Comment (jrieke) [inline content/streamlit-cloud/get-started/deploy-an-app/index.md:36]: Would leave out example 3, looks a bit confusing in my eyes.

**Review by jrieke — APPROVED** [review]
Left few comments on the dir structure, rest LGTM. No need for another review.

File: content/streamlit-cloud/get-started/deploy-an-app/index.md (hunk 3 above)
Comment (sfc-gh-dmatthews) [reply content/streamlit-cloud/get-started/deploy-an-app/index.md]: Thanks.

File: content/streamlit-cloud/get-started/deploy-an-app/index.md (hunk 4 above)
Comment (sfc-gh-dmatthews) [reply content/streamlit-cloud/get-started/deploy-an-app/index.md]: Done. I think I'll add a KB article to my tasks to show how the different combinations of things look (multipage, configuration, Python dependencies, apt-get dependencies, multiple apps in a repository, and apps not in root.

**Pull request merged**